data/core/macros/abilities.cfg

```diff
@@ -415,3 +415,111 @@ Enemy units cannot see this unit while it is in deep water, except if they have
         description= _ "This attack poisons living targets. Poisoned units lose 8 HP every turn until they are cured or are reduced to 1 HP. Poison cannot, of itself, kill a unit."
     [/poison]
 #enddef
+#warning: the abilities below must have affect_adjacent for activate because they are equivalent to weapon specials and affect adjacent unit is the only interest.
```

Comment:
Please remove all the changes to this file. The pull request is for adding the capability to the engine, not for second-guessing what a content author might want to do with that capability. If you need these definitions they should go in your addon, not here in core.